src/main/resources/messages/LibertyBundles.properties

```diff
@@ -9,38 +9,38 @@
+action.io.openliberty.tools.intellij.actions.ViewIntegrationTestReport.text=Liberty: View integration test report
 action.io.openliberty.tools.intellij.actions.ViewIntegrationTestReport.description=Open the Maven Failsafe test report for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.text=View unit test report
+action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.text=Liberty: View unit test report
 action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.description=Open the Maven Surefire test report for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewTestReport.text=View test report
+action.io.openliberty.tools.intellij.actions.ViewTestReport.text=Liberty: View test report
 action.io.openliberty.tools.intellij.actions.ViewTestReport.description=Open the Gradle test report for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=View effective POM
+action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=Liberty: View effective POM
 action.io.openliberty.tools.intellij.actions.ViewEffectivePom.description=Open the pom.xml for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewGradleConfig.text=View Gradle config
+action.io.openliberty.tools.intellij.actions.ViewGradleConfig.text=Liberty: View Gradle config
 action.io.openliberty.tools.intellij.actions.ViewGradleConfig.description=Open the build.gradle for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.text=Refresh Liberty tool window
-action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.description=Refresh Liberty tool window projects
-action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.text=Run Liberty action
+action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.text=Liberty: Refresh tool window
+action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.description=Refresh Liberty tool window Projects
+action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.text=Liberty: Run Liberty action
 action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.description=Run the selected Liberty action
-action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.text=Add Liberty project to the Liberty tool window
+action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.text=Liberty: Add project to the Liberty tool window
 action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.description=Add the selected Liberty project to the Liberty tool window
-action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.text=Remove Liberty project from the Liberty tool window
+action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.text=Liberty: Remove project from the Liberty tool window
```

Comment:
Minor detail but to keep consistent:
Either modify refresh name to "Liberty: Refresh Liberty tool window" or the add/remove actions to "Liberty: Add project to the tool window". 

I'll leave it to what you think sounds best, but suggestion to rename "Liberty: Run Liberty action" -> "Liberty: Run action"